### PR TITLE
Added "jQuery Ajax get and post method"

### PR DESCRIPTION
**Pull request by hasnaas**
with examples and links to official website documentation

### CODE REVIEW

**Comment by texas2010** [comment]
I am not sure why you made other PR for jquery ajax post method. Since you did add jquery ajax post method to this PR first.  https://github.com/freeCodeCamp/guides/pull/554

**Comment by hasnaas** [comment]
This the PR for ajax get method. the other one is for ajax post method

**Comment by texas2010** [comment]
@hasnaas yes I see that. but you did add ajax post method on this PR. that is fine.

**Comment by hasnaas** [comment]
Ooh sorry, you're right. thank you

**Comment by texas2010** [comment]
@hasnaas no problem. all good.

File: src/pages/jquery/jquery-ajax-get-method/index.md [hunk 1]
```diff
@@ -2,14 +2,55 @@
 title: jQuery Ajax Get Method
 ---
 ## jQuery Ajax Get Method
+Sends an asynchronous http GET request to load data from the server. Its general form is :
+~~~~
```
Comment (texas2010) [inline src/pages/jquery/jquery-ajax-get-method/index.md]: use ```javascript

File: src/pages/jquery/jquery-ajax-get-method/index.md [hunk 2]
```diff
@@ -2,14 +2,55 @@
...
+jQuery.get( url [, data ] [, success ] [, dataType ] )
+~~~~
```
Comment (texas2010) [inline src/pages/jquery/jquery-ajax-get-method/index.md]: use ```

File: src/pages/jquery/jquery-ajax-get-method/index.md [hunk 3]
```diff
@@ -2,14 +2,55 @@
...
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/jquery/jquery-ajax-get-method/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+* url : is the only mandatory parameter. This string contains the adress to which to send the request. The returned data will be ignored if no other parameter is specified
+* data : A plain object or string that is sent to the server with the request. 
+* success : A callback function that is executed if the request succeeds.it takes as an argument the returned data. It is also passed the text status of the response.
+* dataType : The type of data expected from the server. The default is Intelligent Guess (xml, json, script, text, html). if this parameter is provided, then the success callback must be provided as well.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+#### Examples
+~~~~
+$.get('http://example.com/resource.json', {category:'client', type:'premium'})
```
Comment (texas2010) [inline src/pages/jquery/jquery-ajax-get-method/index.md]: forget to add ; in the end.

File: src/pages/jquery/jquery-ajax-get-method/index.md [hunk 4]
```diff
@@ -2,14 +2,55 @@
...
+~~~~
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+requests `resource.json` from the server, sending additional data and ignoring the returned result
+~~~~
+$.get('http://example.com/resource.json', {category:'client', type:'premium'}, function(response){ 
+alert("success");
```
Comment (texas2010) [inline src/pages/jquery/jquery-ajax-get-method/index.md]: please use indent.

File: src/pages/jquery/jquery-ajax-get-method/index.md [hunk 5]
```diff
@@ -2,14 +2,55 @@
...
+
+$("#mypar").html(response.amount);
```
Comment (texas2010) [inline src/pages/jquery/jquery-ajax-get-method/index.md]: Please use indent.

**Comment by hasnaas** [comment]
done.

**Comment by texas2010** [comment]
Great!  good job!

**Pull request merged**